ci-operator/step-registry/baremetalds/e2e/ipv4/serial/baremetalds-e2e-ipv4-serial-workflow.yaml

```diff
@@ -0,0 +1,18 @@
+workflow:
+  as: baremetalds-e2e-ipv4-serial
```

Comment:
Maybe it will be better `baremetalds-e2e-serial-ipv4` - considering also that we'll have other configurations (like compact, ipv6, virtualmedia, etc)

Reply:
Although I agree with your naming proposal, aws, gcp use this convention(ends with -serial).

Reply:
Right, but I guess that they are not dealing with multiple configurations as we need?